nvflare/apis/impl/controller.py

```diff
@@ -35,13 +35,14 @@ def __init__(self, task_check_period=0.2):
         self._task_check_period = task_check_period
         self.communicator = None
 
-    def initialize(self, fl_ctx: FLContext):
+    def initialize_controller(self, fl_ctx: FLContext):
```

Comment:
Do we have to change this name? new name seems a bit redundant?

Reply:
reverted to initialize(), fixed minor conflict